src/config.js

```diff
@@ -10,7 +10,7 @@ export default {
   termsOfUseUrl: process.env.TOU_URL || 'https://docs.google.com/document/d/e/2PACX-1vTBNI2Sv5QC8DSBwBL7WHNBdMI-9kPLuN2ev_Y2VDSo-bLeh8qssi7iBv-w0EEQurX9fgFQF4_2lItn/pub',
   commentGuidelinesUrl: process.env.COMMENT_GUIDELINES_URL || 'https://docs.google.com/document/d/e/2PACX-1vTi6AR2Q-PpTNMLTimvdVg8yDuLJ5DURswQ-heCToXj3OwuqNXyt-LIBs-By9znC2A_0HxqlO8vQ_DJ/pub',
   feedbackEmail: process.env.FEEDBACK_EMAIL || 'gogetta@streetlives.nyc',
-  googleMaps: process.env.REACT_APP_GMAPS_URL || 'https://maps.googleapis.com/maps/api/js?key=AIzaSyCFMDqEgQ6VWWJbROzZhHu-f6sktAmTEGU&v=3.exp&libraries=geometry,drawing,places',
+  googleMaps: process.env.REACT_APP_GMAPS_URL || 'https://maps.googleapis.com/maps/api/js?key=AIzaSyABZdHZ1R0cssL9BL9Kx3LpbmDfDJ6oPEc&v=3.exp&libraries=geometry,drawing,places',
```

Comment:
We should probably remove the hard-coded key here as well. Just reference an environment variable with the google maps key, and `assert` that it exists at the top of the module.

Reply:
@shakilhossain1